wpilibc/src/main/native/include/frc/PWM.h

```diff
@@ -186,13 +178,14 @@ class PWM : public wpi::Sendable, public wpi::SendableHelper<PWM> {
    * @param deadbandMin The low end of the deadband pulse width in ms
    * @param min         The minimum pulse width in ms
    */
-  void SetBounds(double max, double deadbandMax, double center,
-                 double deadbandMin, double min);
+  void SetBounds(units::microsecond_t max, units::microsecond_t deadbandMax,
+                 units::microsecond_t center, units::microsecond_t deadbandMin,
+                 units::microsecond_t min);
```

Comment:
The doc comment references units of ms- Should this be changed to microseconds or μs?